boards/arm/gd32f450v_start/board.cmake

```diff
@@ -0,0 +1,7 @@
+# Copyright (c) 2022, Teslabs Engineering S.L.
+# SPDX-License-Identifier: Apache-2.0
+
+board_runner_args(jlink "--device=GD32F450VK" "--speed=4000")
+
+include(${ZEPHYR_BASE}/boards/common/openocd.board.cmake)
```

Comment:
It may be good to use pyOCD rather than OpenOCD.

Reply:
in this case openocd works fine (using stm32 configs really), so it doesn't hurt adding support for it. I'll also add pyocd.

Reply:
I got it. Let's keep it.